lib/round.rb

```diff
@@ -0,0 +1,69 @@
+class Round
+  attr_reader :deck, 
+              :turns, 
+              :current_turn, 
+              :number_correct, 
+              :correct_qs, 
+              :curr_card
```

Comment:
You don't need this `attr_reader` because you never access `@curr_card` outside of this class so you can call `@curr_card` directly.